### PR TITLE
log error when setting timezone to UTC fails

### PR DESCRIPTION
**Pull request by butonic**
yes, we had this kind of problem. we should log it as an error at least.

### CODE REVIEW

**Comment by DeepDiver1975** [comment]
:+1:

**Comment by butonic** [comment]
@PhilippSchaffrath quick one line review pls

**Comment by butonic** [comment]
@PhilippSchaffrath nevermind

File: lib/base.php [hunk 1]
```diff
@@ -537,7 +537,9 @@ public static function init() {
 		@ini_set('display_errors', 0);
 		@ini_set('log_errors', 1);
 
-		date_default_timezone_set('UTC');
+		if(!date_default_timezone_set('UTC')) {
```
Comment (PhilippSchaffrath) [inline lib/base.php]: I am not sure how this will ever be evaluated to true, since 'UTC' is definitely a valid value. Is there any edge-case i am missing?

**Pull request merged**